manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```
@@ -378,6 +378,12 @@ data:
     #
     # owners: (@openshift/networking)
     #
+    # ovnkube_master_egress_routing_via_host" informs (1 or 0) if the OVN-K cluster has
+    # enabled `routingViaHost` gateway mode or not.
+    - '{__name__="ovnkube_master_egress_routing_via_host"}'
```

If we forward the "raw" metric, it will have labels like `instance`, `pod` that aren't necessary. The good practice would be to record a new metric that aggregates away all labels and send the recorded metric.

In practice, you need to update https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/common/alert-rules.yaml like this:

```
spec:
  groups:
  - ...
  - name: cluster-network-operator-ovn.rules
    rules:
    - record: cluster:ovnkube_master_egress_routing_via_host:max
      expr: max(ovnkube_master_egress_routing_via_host)

```

```suggestion
    - '{__name__="cluster:ovnkube_master_egress_routing_via_host:max"}'
```

---

oh thanks so much! makes sense, let me see if I can get that into CNO first.

---

I have pushed this into https://github.com/openshift/cluster-network-operator/pull/1425